runhouse/main.py

```diff
@@ -128,14 +129,47 @@ def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluste
     c.ssh()
 
 
-def _print_status(config):
+###############################
+# Status helping functions
+###############################
+
+
+def _adjust_resource_type(resource_type: str):
+    """
+    status helping function. transforms a str form runhouse.resources.{X.Y...}.resource_type to runhouse.resource_type
+    """
+    if "runhouse" not in resource_type:
+        return resource_type
+    resource_type = resource_type.split(".")
+    resource_type = ".".join([resource_type[0], resource_type[-1]])
+    return resource_type
+
+
+def _resource_name_to_rns(name: str, current_cluster: Cluster):
```

Comment:
would using `rns_client.resolve_rns_path` suffice here?

Reply:
Not really. The name passed as an arg is the object store key, which is in the 'name_of_resource' format. It does not contain the special characters that are checked in `rns_client.resolve_rns_path`. Therefore the name will be returned as passed, not as `rns_address`, even if the resource has one.